packages/components/Molecules/Form/stories.js

```diff
@@ -6,90 +6,135 @@ import {
   withKnobs,
   text,
   boolean,
-  object
-} from '@storybook/addon-knobs/vue'
+  object,
+  array
+} from '@storybook/addon-knobs'
+
+const selectItems = object('Values', [
+  { id: 'vic', name: 'Vic' },
+  { id: 'nsw', name: 'New South Wales' },
+  { id: 'wa', name: 'Western Australia' },
+  { id: 'ql', name: 'Queensland' }
+])
```

Comment:
Hey @rubyllamadora I think something is wrong with this one, I got it injected into other stories. 
https://storybook-ripple-pr-765.lagoon.vicsdp.amazee.io/?path=/story/atoms-button--primary
![Screen Shot 2020-09-04 at 5 31 07 pm](https://user-images.githubusercontent.com/31641325/92211852-71538d80-eed4-11ea-890c-10c69956f4df.png)

Reply:
Hmm I can actually replicate on first load of the page. Looks like it's being taken as story-wide global variable. Will look into this.

Reply:
I have put it back to it's own story. Thanks for the review, good spotting @tim-yao